swift/ql/test/library-tests/hidden-ast/Conversions.expected

```diff
@@ -0,0 +1,3 @@
+| chained_conversion.swift:2:12:2:14 | (Int) ... | LoadExpr | chained_conversion.swift:2:12:2:14 | .self | DotSelfExpr |
+| chained_conversion.swift:2:12:2:14 | (Int?) ... | InjectIntoOptionalExpr | chained_conversion.swift:2:12:2:14 | (Int) ... | LoadExpr |
+| chained_conversion.swift:2:12:2:14 | .self | DotSelfExpr | chained_conversion.swift:2:12:2:12 | x | DeclRefExpr |
```

Comment:
How would this have been different before the change? (I'm not very familiar with the hidden AST / which elements are hidden)

Reply:
Before the change, they would all have returned the fully resolved `x` as their sub-expr. Now, you get the full chain without going through the Immediate accessors.

Reply:
Exactly.